src/Analyzers/VisualBasic/Tests/MakeDeclarationPartial/MakeDeclarationPartialTests.vb

```diff
@@ -0,0 +1,179 @@
+﻿' Licensed to the .NET Foundation under one or more agreements.
+' The .NET Foundation licenses this file to you under the MIT license.
+' See the LICENSE file in the project root for more information.
+
+Imports Microsoft.CodeAnalysis.CodeFixes
+Imports Microsoft.CodeAnalysis.Diagnostics
+Imports Microsoft.CodeAnalysis.Editor.VisualBasic.UnitTests.Diagnostics
+Imports Microsoft.CodeAnalysis.VisualBasic.MakeDeclarationpartial
+
+Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.UnitTests.MakeDeclarationPartial
+    <Trait(Traits.Feature, Traits.Features.CodeActionsMakeDeclarationPartial)>
```

Comment:
use new form.

Reply:
Is this one about new test infra or what (asking because you already pointed to the `SuppressMessage` of VB side above)? Is this one already resolved with current PR state?